test/app_profiler/profile_server_test.rb

```diff
@@ -116,10 +116,10 @@ class ProfileServerTest < ProfileServerTestCase
           lines = response.lines
           assert(lines.shift.match?(/HTTP.*200/))
           assert_equal("Content-Type: application/json\r\n", lines.shift)
-          assert_equal("Access-Control-Allow-Origin: *\r\n", lines.shift)
           length_line = lines.shift
           assert(length_line =~ (/Content-Length: (.*)/))
           content_length = Regexp.last_match(1).to_i
+          assert_equal("Access-Control-Allow-Origin: *\r\n", lines.shift)
```

Comment:
order needed to move since this test is picky about the order it receives the headers in, and I changed it above to keep the linter happy